### PR TITLE
Feature/frontend/page/verify email

### PR DESCRIPTION
**Pull request by rem1niscence**

### CODE REVIEW

File: frontend/src/core/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import { Col, Row } from "react-bootstrap";
+import HelpLink from "./HelpLink";
+
+class NavBar extends Component {
```
Comment (xerp) [inline frontend/src/core/components/NavBar.js]: The file and class name should be **Navbar**.

File: frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      "What was the name of your first stuffed animal?",
+      "In what city or town did your mother and father meet?",
+      "Where were you when you had your first kiss?",
+      "What is the first name of the boy or girl that you first kissed?",
+      "What was the last name of your third grade teacher?",
+      "In what city does your nearest sibling live?",
+      "What is your maternal grandmother's maiden name?",
+      "In what city or town was your first job?",
+      "What is the name of the place your wedding reception was held?"
+    ]
```
Comment (xerp) [inline frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js]: We need create a process to insert this data on db.
And then that backend send back these questions.

File: frontend/src/views/Auth/VerifyEmail/VerifyEmail.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <NavBar />
+        <Row>
+          <Col
+            id="main"
+            md={{ span: 8, offset: 2 }}
+            lg={{ span: 4, offset: 4 }}
+          >
+            <h1>
+              We send an email to this address <br />
+              ******23@gmail.com
```
Comment (xerp) [inline frontend/src/views/Auth/VerifyEmail/VerifyEmail.js]: This value is returned by the backend when sign up is success.

**Review by xerp — CHANGES_REQUESTED** [review]
@justaname94 Please see the comments about the review.

File: frontend/src/views/Auth/VerifyEmail/VerifyEmail.js (hunk 3 above)
Comment (rem1niscence) [reply frontend/src/views/Auth/VerifyEmail/VerifyEmail.js]: You mean the backend will return the formatted email?

File: frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js (hunk 2 above)
Comment (rem1niscence) [reply frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js]: Yes, this was made only as a placholder until the backend was ready

Comment (xerp) [reply frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js]: Ok

File: frontend/src/views/Auth/VerifyEmail/VerifyEmail.js (hunk 3 above)
Comment (xerp) [reply frontend/src/views/Auth/VerifyEmail/VerifyEmail.js]: I meant to the raw email, but, yes, it is a good idea.

File: frontend/src/core/components/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import { Col, Row } from "react-bootstrap";
+import HelpLink from "./HelpLink";
+
+class NavBar extends Component {
```
Comment (xerp) [inline frontend/src/core/components/Navbar.js]: You left the class name

File: frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  state = {
+    securityQuestions: []
+  };
+
+  componentDidMount = () => {
+    const securityQuestions = ["Select one", ...this.state.securityQuestions];
+    this.setState({ securityQuestions });
+  };
+
+  handleChange = e => {
```
Comment (xerp) [inline frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js]: This is a class method, You need to change to the non-anonymous function style.

**Review by xerp — CHANGES_REQUESTED** [review]
Two changes are remaining.

**Review by xerp — APPROVED** [review]
CR approved.

**Pull request merged**